tinygrad/codegen/uops.py

```diff
@@ -59,6 +59,10 @@ def lt(self, x): return UOp.alu(BinaryOps.CMPLT, self, ufix(self.dtype, x))
   def ge(self, x): return -self.lt(x)
   def max(self, x): return UOp.alu(BinaryOps.MAX, self, x)
   def min(self, x): return -UOp.alu(BinaryOps.MAX, -self, -x)
+
+  # symbolic min max
+  def maxval(self): return self.src[1] if self.op is UOps.DEFINE_VAR and len(self.src) == 2 else self
```

Comment:
these are also needed to replace other Nodes, not just Variable.

Reply:
Yea, I upstreamed "divides", I would expect it to be similar. Also, I would expect the return type of this to be a number.